### PR TITLE
Added skeleton

### PR DESCRIPTION
**Pull request by sergeycheremisin**

### CODE REVIEW

**Review by it-volunteers-ukraine — APPROVED** [review]

File: client/pages/RootLayout.tsx
Comment (it-volunteers-ukraine) [inline client/pages/RootLayout.tsx:1]: it will be component

File: client/pages/dashboard/index.tsx
Comment (it-volunteers-ukraine) [inline client/pages/dashboard/index.tsx:1]: why empty?

**Review by it-volunteers-ukraine — APPROVED** [review]

**Review by rpkyrych — APPROVED** [review]

File: client/pages/dashboard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from "react"
```
Comment (Wolshebnik) [inline client/pages/dashboard/index.tsx:1]: Delete pls

File: client/pages/index.tsx [hunk 2]
```diff
@@ -1,118 +1,9 @@
-import Image from 'next/image'
-import { Inter } from 'next/font/google'
+import React from "react"
```
Comment (Wolshebnik) [inline client/pages/index.tsx:1]: Delete pls

File: client/components/RootLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from "react"
```
Comment (Wolshebnik) [inline client/components/RootLayout.tsx:1]: Delete pls

File: client/components/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from "react"
```
Comment (Wolshebnik) [inline client/components/Header.tsx:1]: Delete pls

File: client/components/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from "react"
```
Comment (Wolshebnik) [inline client/components/Footer.tsx:1]: Delete pls

File: client/components/RootLayout.tsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+import {Header} from "@/components/Header"
+import {Footer} from "@/components/Footer"
```
Comment (Wolshebnik) [inline client/components/RootLayout.tsx:4]: Create import in components
just use import from components

File: client/pages/index.tsx [hunk 7]
```diff
@@ -1,118 +1,9 @@
...
 
-const inter = Inter({ subsets: ['latin'] })
-
-export default function Home() {
+export default function HomePage(): React.ReactNode {
```
Comment (Wolshebnik) [inline client/pages/index.tsx:3]: Delete React.ReactNode doesn't need it
or use JSX.Element

File: client/pages/dashboard/index.tsx [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+
+export default function DashboardPage(): React.ReactNode {
```
Comment (Wolshebnik) [inline client/pages/dashboard/index.tsx:3]: Delete React.ReactNode doesn't need it
or use JSX.Element

File: client/components/Footer.tsx [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+
+export const Footer = (): React.ReactNode => {
```
Comment (Wolshebnik) [inline client/components/Footer.tsx:3]: Delete React.ReactNode doesn't need it
or use JSX.Element

File: client/components/Header.tsx [hunk 10]
```diff
@@ -0,0 +1,9 @@
...
+
+export const Header = (): React.ReactNode => {
```
Comment (Wolshebnik) [inline client/components/Header.tsx:3]: Delete React.ReactNode doesn't need it
or use JSX.Element

File: client/components/RootLayout.tsx [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+interface RootLayoutProps {
+  children: React.ReactNode
+}
+
+export const RootLayout = ({ children }: RootLayoutProps): React.ReactNode => {
```
Comment (Wolshebnik) [inline client/components/RootLayout.tsx:10]: Delete React.ReactNode doesn't need it
or use JSX.Element

File: client/pages/_app.tsx [hunk 12]
```diff
@@ -1,6 +1,12 @@
 import '@/styles/globals.css'
 import type { AppProps } from 'next/app'
 
+import { RootLayout } from "@/components/RootLayout"
```
Comment (Wolshebnik) [inline client/pages/_app.tsx:4]: Create import in components
just use import from components

**Pull request closed without merging**